app/app.star

```diff
@@ -0,0 +1,39 @@
+constants = import_module("/constants/constants.star")
+
+
+def run(plan, postgres_db, postgress_password):
+    postgres_port = "{}".format(postgres_db.service.ports["postgresql"].number)
+
+    container_cmd = "./db-script.sh && python3 manage.py runserver 0.0.0.0:{}".format(
+        constants.BACKEND_SERVICE_HTTP_PORT
+    )
+
+    service_config_build = ServiceConfig(
+        image=ImageBuildSpec(
+            image_name=constants.BACKEND_SERVICE_IMAGE_NAME,
```

Comment:
I wonder if this should just be a constant in this file; for a new person they have to otherwise navigate(maybe this is scarier than a normal string, constant in this file) and I am not sure if the symbols are clickable

Reply:
Probably you are right, could be less scary just to read the strings here. I will do the change.

Reply:
done